docs/snippets/start-connect-dydx-snippet.mdx

```diff
@@ -1,49 +1,74 @@
-The dYdX blockchain requires some additional steps to start Connect.
+**The minimum required version for Connect with dYdX is `v1.0.5+`.**
 
-**The minimum required version for Connect with dYdX is `v1.0.1+`.**
-
-First, please ensure you've received your API keys for Raydium markets. If you have not received API keys, please reach out to the team in the relevant channels.
+First, please ensure you've received your **API keys** for Raydium markets. If you have not received API keys, please reach out to the team in the relevant channels.
 
 Next, place your API keys under their corresponding URLs in the following file and save it to your system. Keep the file path handy as we will pass it into a flag when running Connect.
```

Comment:
What file path is this referring to?

Reply:
first part of the sentence: "Next, place your API keys under their corresponding URLs in the following file and save it to your system."